src/platform/i386/boot_comp.c

```diff
@@ -155,8 +158,10 @@ kern_boot_comp(void)
 	 * Need to account for the pages that will be allocated as
 	 * PTEs
 	 */
+	local_pgd = (pgtbl_t)chal_va2pa(mem_boot_alloc(1));
+	if (pgtbl_activate(ct, BOOT_CAPTBL_SELF_CT, BOOT_CAPTBL_SELF_LOCAL_PT, local_pgd, 0)) assert(0);
 	nkmemptes = boot_nptes(mem_utmem_end() - mem_boot_end());
-	ret = boot_pgtbl_mappings_add(ct, pgtbl, BOOT_CAPTBL_KM_PTE, "untyped memory", mem_boot_nalloc_end(nkmemptes),
+	ret = boot_pgtbl_mappings_add(ct, BOOT_CAPTBL_SELF_LOCAL_PT, BOOT_CAPTBL_KM_PTE, "untyped memory", mem_boot_nalloc_end(nkmemptes),
```

Comment:
OK, I think i get the point of all of this.

Aside from the obvious issues with the commit (types, unused local variables, etc...), I think this is OK.  I don't like using `local` in code that is supposed to be generic (i.e. not only used for *your* functionality).  In the enum of capabilities, perhaps we can use `BOOT_CAPTBL_SELF_UNTYPED_PT`?